database-commons/src/main/java/io/cdap/plugin/db/DBErrorDetailsProvider.java

```diff
@@ -76,7 +78,8 @@ private ProgramFailureException getProgramFailureException(SQLException e, Error
         externalDocumentationLink);
     }
     return ErrorUtils.getProgramFailureException(new ErrorCategory(ErrorCategory.ErrorCategoryEnum.PLUGIN),
-      errorMessage, errorMessageWithDetails, getErrorTypeFromErrorCode(errorCode), false, e);
+      errorMessage, errorMessageWithDetails, getErrorTypeFromErrorCode(errorCode), false, ErrorCodeType.SQLSTATE,
+      sqlState, DBUtils.MYSQL_SUPPORTED_DOC_URL, e);
```

Comment:
The supported doc url should come from child class, we cannot hardcoded mysql.

Reply:
🤦  typo

---

Tested Again

![image](https://github.com/user-attachments/assets/0c44b406-473e-4479-ba23-f79e4aa86fb5)